### ISSUE TITLE
Incorporate date in Press Release collection download links to resolve link title ambiguity 

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/onrr.gov-site#3220]
In the press release collection (current and archived), the links to download the press release document are all the same and run into accessibility issues due to ambiguous link titles:
<img width="496" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1BvQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--21ae91c97afe2df5c7312e58166ddbbe5ff388c0/image.png" alt="image.png" />

None of the press release documents have clean titles in the CMS, and there is no way for me to modify links within the collection function:
<img width="1204" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2ZvQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7ea1a68625fe2fd19f33817f3be8c9bbaba605a6/image.png" alt="image.png" />
<img width="746" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3ZvQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--04efb473579bee6a0c5060bcb45840bff6bdf777/image.png" alt="image.png" />

I think the easiest way to resolve ambiguity would be to incorporate the publication date into the download link.
Instead of just "View press release document" this could be reprogramed to say "View March 29, 2024 press release document"
<img width="381" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3JvQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--9bd55f80b33810cea70b8730cedde35314895c9f/image.png" alt="image.png" />

**Comment by greensomes** [comment]
@cthomasONRR, I did some research and found some additional options we have to resolve this accessibility violation type. Rather than adding the date to the link, we can either provide an aria-label to differentiate each link, or provide additional context within the link text that is only available to screen readers and is hidden visually. Both options achieve the same result of providing additional context for screen readers while also retaining the clean visuals of the simple link text. What do you think?

**Comment by cthomasONRR** [comment]
@greensomes Thanks for doing this research! Could we incorporate the dates within the link text that is only available to screen readers (and hidden visually)? I think the dates are the only thing standardized in this collection... the file names and CMD doc names are a mess.

**Comment by greensomes** [comment]
@cthomasONRR, you got it!!
